index.html

```diff
@@ -0,0 +1,503 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Challenge</title>
+    <link rel="stylesheet" href="/Styles/style.css" />
+    <link rel="stylesheet" href="/Styles/reset.css" />
+    <link rel="stylesheet" href="/Styles/layout.css" />
+    <link rel="stylesheet" href="/Styles/responsive.css" />
+  </head>
+  <body>
+    <div class="body-container three-grid-column__big-two centered">
+      <!-- *** LEFT BAR *** -->
+      <aside class="left-bar">
+        <div class="left-bar-wrapper centered grid-auto-flow__row">
+          <img
+            src="/Assets/Logo/twitter-logo.png"
+            alt="Twitter logo"
+            class="logo"
+          />
+          <!-- menu bar -->
+          <div class="menu-container">
+            <ul>
+              <li class="menu-bar">
+                <a href="" class="two-grid-column__expand-two">
+                  <img src="/Assets/Icons/home.png" alt="home icon" />
+                  <h2><span>Home</span></h2></a
+                >
+              </li>
+              <li class="menu-bar">
+                <a href="" class="two-grid-column__expand-two">
+                  <img src="/Assets/Icons/hash-tag.png" alt="hashtag icon" />
+                  <h2>Explore</h2></a
+                >
+              </li>
+              <li class="menu-bar">
+                <a href="" class="two-grid-column__expand-two">
+                  <img src="/Assets/Icons/bell.png" alt="bell icon" />
+                  <h2>Notifications</h2></a
+                >
+              </li>
+              <li class="menu-bar">
+                <a href="" class="two-grid-column__expand-two">
+                  <img src="/Assets/Icons/email.png" alt="message icon" />
```

Comment:
great use of the classes next time maybe instead of resizing download smaller icons